src/core/agents.jl

```diff
@@ -17,6 +17,19 @@ and hence it is the **the only supported way to create agent types**.
 """
 abstract type AbstractAgent end
 
+__AGENT_GENERATOR__ = Dict{Symbol, Expr}()
+
+"""
+    NoSpaceAgent <: AbstractAgent
+The minimal agent struct for usage with `nothing` as space (i.e., no space).
```

Comment:
I am confused. Why is this docstring here? Shouldn't it be at No Space AGent...?

Reply:
Not to repeat again the same thing I evaluated the NoSpaceAgent expression in __AGENT_GENERATOR to create NoSpaceAgent